feed-creator.js

```diff
@@ -33,8 +34,8 @@ module.exports = function (meta, data) {
 
       feed.addItem({
         title: item.title,
-        id: `https://bkardell.com/bcd-watch/${item.file}`,
-        link: `https://bkardell.com/bcd-watch/${item.file}`,
+        id: `https://bcd-watch.igalia.com/${item.file}`,
```

Comment:
```suggestion
        id: `https://bcd-watch.igalia.com/${data.path}/${item.file}`,
```

Reply:
`${data.path}` should be `${directory}` (see lines 19-21).

Reply:
no directory is "/out/" whereas this is the particular subdirectory (weekly or weekly-completed) - but you're right, those should change too - I made those suggestions as well

Reply:
No, see line 7: I made directory a local variable where I filtered the “out” part (plus a little more) out of `data.path` in order to not have the URLs look bonkers, and maybe break.